cmd/all-in-one/main.go

```diff
@@ -114,7 +114,7 @@ by default uses only in-memory database.`,
 			if err != nil {
 				logger.Fatal("Failed to create span writer", zap.Error(err))
 			}
-			dependencyReader, err := storageFactory.CreateDependencyReader()
+			dependencyReader, err := v2Factory.CreateDependencyReader()
```

Comment:
@yurishkuro for v1, the factory adapter is create directly so we have direct access to this function - is this okay?

Reply:
yes